backend/src/loader.ts

```diff
@@ -15,16 +15,20 @@ import { publisherRoutes } from "./routes/publisher.route"
 import { titleRoutes } from "./routes/title.route"
 import { userRoutes } from "./routes/user.route"
 import { logError } from "./util/errorHandling"
+import { adminRoutes } from "./routes/admin.route"
+import AWS from "aws-sdk"
+import process from "process"
 
 const UNCAUGHT_EXCEPTION_EXIT_STATUS = 1
 
 const appRouter = (): Router => {
   const router = Router()
 
-  router.use("/publisher", authenticate, publisherRoutes)
-  router.use("/issue", authenticate, issueRoutes)
-  router.use("/title", authenticate, titleRoutes)
+  router.use("/publisher", authenticate(), publisherRoutes)
+  router.use("/issue", authenticate(), issueRoutes)
+  router.use("/title", authenticate(), titleRoutes)
   router.use("/user", userRoutes)
+  router.use("/admin", authenticate(true), adminRoutes)
```

Comment:
Maybe something like this is better?
```suggestion
  const [authAdmin, authRegular] = [authenticate(true), authenticate()];
  
  router.use("/publisher", authRegular, publisherRoutes)
  router.use("/issue", authRegular, issueRoutes)
  router.use("/title", authRegular, titleRoutes)
  router.use("/user", userRoutes)
  router.use("/admin", authAdmin, adminRoutes)
```